### DISCUSSION TITLE
[BUG]v4.9 JSQLParser Version : can transfer Select to PlainSelect, how can i get the tableName or where,thanks

### DISCUSSION THREAD

**Discussion by juliusMEI** [JSQLParser/JSqlParser#1990] [Q&A]
4.6-4.8 可以把Select转为PlainSelect,4.9怎么转呢

**Comment by juliusMEI** [comment]
![image](https://github.com/JSQLParser/JSqlParser/assets/79579168/9d04a1e5-25c5-40db-bba9-5764e8eeb1cd)

**Comment by manticore-projects** [comment]
Please see the documentation here: https://jsqlparser.github.io/JSqlParser/usage.html#parse-a-sql-statement
